container/20221114.md

```diff
@@ -0,0 +1,48 @@
+# 2022-11-14
+## Participants (github handles preferred)
+* @master-caster
+* @garloff
+* @matfechner
+* @jnull
+* @o-otte
+* @joshmue
+* @horazont
+* @ajfriesen
+* @fkr
+* @ComradeOgilvy
+* @mxmxchere
+* @alexander-diab
+* @janiskemper
+* @lossack
+## Summary & Minutes transfer
+@joshmue
+
+## Agenda
+
+* Standardizing cluster properties and parameters https://github.com/SovereignCloudStack/issues/issues/181
+    * Progress #212: https://github.com/SovereignCloudStack/issues/issues/212 - 2 Test Scripts available, not yet perfect: kubeapi reports LB to be up but takes another minute before it works
+* RWX Storage discussion: How to implement? Mandatory? Standardize at least? What are the use cases?
+  1. HA app that wants the filesystem to do the hard work of consistent replication - LEGACY (e. g. old LAMP)
+  2. Users that don´t understand RWO can be used if pods are on the same node
+  3. Bare Metal nodes
+  4. Generally: Sharing data between pods
+  5. Generally: Sharing application code between pods (often used when LAMP apps are transitioned to k8s)
+* Implementation options
+  - NFS
+  - cephfs
+  - manila: ganesha NFS, via userspace service (not fuse) (https://github.com/nfs-ganesha/nfs-ganesha/wiki)
+* PoC for SCS operator from 23tech: 
+    - Minutes PR (https://github.com/SovereignCloudStack/minutes/pull/74) is wrong place for discussion, so will open new issue (AI @garloff)
+    - Would be nice to have PR reviewed
+* Conformance test framework discussion. Two different main scenarios:
+    1. Test creation/changing/... clusters via SCS standardized interfaces (the REST interface we are discussing)
+       - (Cluster API specific): Cluster API test framework.
+         * https://github.com/syself/cluster-api-provider-hetzner/tree/main/test/e2e
+         * https://github.com/syself/cluster-api-provider-hetzner/blob/main/test/e2e/conformance_test.go
+         * https://github.com/kubernetes-sigs/cluster-api/blob/fa70f61a77828cea2d5d059ffd0dfcf6bcbc9f2d/test/e2e/k8s_conformance.go#L109
```

Comment:
As far as *SCS conformance* tests are concerned, we can not assume that the implementation uses CAPI.
(That said, the tests that are part of the CAPI tests may not make that assumption everywhere ... at least some comment from Janis indicated so.)

Reply:
Moved that part to the new 3rd scenario.